plans/install.pp

```diff
@@ -111,7 +122,7 @@
 
   # Upload the pe.conf files to the hosts that need them
   pe_xl::file_content_upload($master_pe_conf, '/tmp/pe.conf', $master_host)
-  pe_xl::file_content_upload($puppetdb_database_pe_conf, '/tmp/pe.conf', $puppetdb_database_host)
+  pe_xl::file_content_upload($puppetdb_database_pe_conf, '/tmp/pe.conf', $database_target)
```

Comment:
This line looks like it would overwrite the master pe.conf if the database target was the master...   Is that intended?

Reply:
database_target will be an empty array in the large deploy. So this command will not have any effect.

Reply:
database_target will be an empty array in the large deploy. So this command will not have any effect.